libs/pipelines/api_pipeline.py

```diff
@@ -179,6 +183,37 @@ def build_counties_timeseries(
     return APIOutput(key, county_api_data, intervention)
 
 
+def build_county_summary_from_model_output(input_dir) -> List[APIOutput]:
+    """Builds lists of counties available from model output.
+
+    Args:
+        input_dir: Input directory.  Should point to county output.
+
+    Returns: List of API Output objects.
+    """
+    found_fips_by_state = defaultdict(set)
+    found_fips = set()
+    for path in pathlib.Path(input_dir).iterdir():
+        if not str(path).endswith('.json'):
+            continue
+
+        state, fips, intervention, _ = path.name.split('.')
```

Comment:
also does python have the like match stuff (I'm thinking in javascript right now), but like a regex match?

Reply:
yeah it does - could do it later but this works well enough for right now